source/developer/fx-guidelines.rst

```diff
@@ -45,16 +45,16 @@ Forgiving extends to compatibility as well, and whether you’re working from a
 
```

Comment:
"Fast has two parts: being _responsive_ and being _quick_" doesn't seem to be rendering on github or docs.mattermost.